relevanceai/http_client.py

```diff
@@ -99,3 +101,102 @@ def check_auth(self):
             return response.status_code == 200    
         except:
             raise Exception("Invalid auth details.")
+
+    def kmeans_cluster(self,
+        dataset_id: str,
+        vector_fields: list,
+        filters: List = [],
+        k: Union[None, int] = 10,
+        init: str = "k-means++",
+        n_init: int = 10,
+        max_iter: int = 300,
+        tol: float = 1e-4,
+        verbose: bool = True,
+        random_state = None,
+        copy_x: bool = True,
+        algorithm: str ="auto",
+        alias: str = "default",
+        cluster_field: str="_cluster_",
+        overwrite: bool = False
+    ):
+        """
+        This function performs all the steps required for Kmeans clustering:
+        1- Loads the data
+        2- Clusters the data
+        3- Updates the data with clustering info
+        4- Adds the centroid to the hidden centroid collection
+
+        Parameters:
```

Comment:
I believe it should be:
```
Parameters
---------------
```

If this is like this anywhere else, please update too.